app/forms/field_handlers/field_handler.py

```diff
@@ -18,24 +21,41 @@ class FieldHandler(ABC):
     def __init__(
         self,
         answer_schema: dict,
-        error_messages: dict = None,
+        schema: QuestionnaireSchema,
         answer_store: AnswerStore = None,
+        list_store: ListStore = None,
         metadata: dict = None,
```

Comment:
I thought we agreed we will now always pass everything to generate_form > field handlers. Field handler should now always get passed the stores, and metadata. Atm, the Value source resolver still doesn't get what it needs as an empty store is initialised when the value is None which is just hiding the problem, we should pass the actual stores in through from the route.

Reply:
Yeah, remember that bit when I said I was confused what we were doing, and it was never really cleared up.